### PR TITLE
Link patterns support + project structure refactoring

### PR DESCRIPTION
**Pull request by Bakanych**
- Implements feature #9 
- code refactoring
- allure.commons nuget package now supports both core 2.0 and net 4.6.1 targets

### CODE REVIEW

File: Allure.SpecFlowPlugin/PluginConfiguration.cs [hunk 1]
```diff
@@ -1,79 +1,133 @@
...
+    //        TestMethodRegex = ParseRegex((string)grouping["packages"]["testMethod"]);
 
-            }
-            catch (Exception)
-            {
-                return null;
-            }
-        }
-    }
+    //        var labels = sf["labels"];
```
Comment (baev) [inline Allure.SpecFlowPlugin/PluginConfiguration.cs]: whats the point of keeping the code commented?

File: Allure.SpecFlowPlugin/allureConfig.json [hunk 2]
```diff
@@ -1,6 +1,11 @@
 ﻿{
   "allure": {
-    "directory": "allure-results"
+    "directory": "allure-results",
+    "links": [
+      "{link}",
```
Comment (baev) [inline Allure.SpecFlowPlugin/allureConfig.json]: how is that works?

File: Tests.SpecRun/TestData/Labels.feature.cs [hunk 3]
```diff
@@ -115,12 +115,14 @@ public virtual void V1_2Api_CoreApiTest1()
         
         [TechTalk.SpecRun.ScenarioAttribute("[v1.2] [api.core.create] Api test 2", new string[] {
                 "api",
-                "create"}, SourceLine=10)]
+                "create",
+                "link:http://google.com"}, SourceLine=10)]
```
Comment (baev) [inline Tests.SpecRun/TestData/Labels.feature.cs]: can we use `example.org` links for examples?

File: Allure.SpecFlowPlugin/PluginConfiguration.cs (hunk 1 above)
Comment (Bakanych) [reply Allure.SpecFlowPlugin/PluginConfiguration.cs]: Massive code review drawback ;( Fixed. Thanks!

File: Allure.SpecFlowPlugin/allureConfig.json (hunk 2 above)
Comment (Bakanych) [reply Allure.SpecFlowPlugin/allureConfig.json]: I put description in wiki already: https://github.com/allure-framework/allure-csharp/wiki/Allure.Commons

File: Tests.SpecRun/TestData/Labels.feature.cs (hunk 3 above)
Comment (Bakanych) [reply Tests.SpecRun/TestData/Labels.feature.cs]: Sure. Fixed too.

File: Allure.SpecFlowPlugin/allureConfig.json (hunk 2 above)
Comment (Bakanych) [reply Allure.SpecFlowPlugin/allureConfig.json]: usage example: https://github.com/allure-framework/allure-csharp/wiki/SpecFlow-Adapter#links

**Pull request merged**